### PR TITLE
[AI-0001]: 💄 feat: one page, login page, register page build

### PR DESCRIPTION
**Pull request by kinga-kus**
# Description
In this PR I have added the OnePage (it will be modified), LoginPage and RegisterPage component.

### Screenshoots

https://user-images.githubusercontent.com/103138106/230653630-fd64d457-055c-4d13-9f0e-c076236f883e.mov

Zrzut ekranu 2023-04-7 o 19 43 45

Zrzut ekranu 2023-04-7 o 19 43 13

### CODE REVIEW

File: .DS_Store
Comment (adam-kostuch) [inline .DS_Store:1]: If it's possible please remove this file from PR, to do so follow the steps: 
```
1. git reset --soft HEAD~1 - command use to retrieve locally yours recent changes
2. git checkout .DS_Store - I don't remember if it's this command or git rm .DS_Store, but I do it manually from git extension in vscode, clicking minus sign right next to file name (and I recommend you this idea)
3. git add . && git commit -m "..." - once again add those files
4. git push --force - the force here is needed as you pulled back some changes
```

After all of that, go to `./client/.gitignore` and wherever paste the `.DS_Store` to don't add it again.

File: client/src/HomeNavbar.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Box, Typography } from '@mui/material';
+
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+const HomeNavbar = () => {
+  return (
+    <Box sx={{ position: 'absolute' }}>
```
Comment (adam-kostuch) [inline client/src/HomeNavbar.tsx]: Slight suggestion, nothing necessary. Next time try to write it as shown below, it's dedicated way to do it by MUI and we should follow it.
```suggestion
    <Box position="absolute">
```

File: client/src/HomeNavbar.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          variant="h6"
+          component="div"
+          sx={{
+            marginLeft: '100px',
+            marginTop: '50px',
+            color: 'white',
+            flexGrow: 1,
+            display: { xs: 'none', sm: 'block' },
+          }}
+        >
```
Comment (adam-kostuch) [inline client/src/HomeNavbar.tsx]: Same thing as above, it's cleaner to set margins as shown here
```suggestion
        <Typography
          variant="h6"
          component="div"
          ml="100px"
          mt="50px"
          sx={{
            color: 'white',
            flexGrow: 1,
            display: { xs: 'none', sm: 'block' },
          }}
        >
```

File: client/src/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+                </Box>
+                <Link to="/register" style={{ textDecoration: 'none' }}>
+                  <Typography sx={{ color: '#FF6700' }}>
+                    Create an account
+                  </Typography>
+                </Link>
+              </Grid>
+              <Grid container spacing={2}>
+                <Grid item xs={12}>
+                  <CustomBorderTextField
```
Comment (adam-kostuch) [inline client/src/LoginPage/LoginPage.tsx:117]: FYI. We probably should use `Formik` package here for field validation.

File: client/src/RegisterPage/CustomSlider.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            }}
+          >
+            <Typography variant="h4">Welcome!</Typography>
+            <Typography>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            </Typography>
+          </Box>
+        </SwiperSlide>
+      </Swiper>
```
Comment (adam-kostuch) [inline client/src/RegisterPage/CustomSlider.tsx]: I think this works better.

```suggestion
      <Swiper
        spaceBetween={30}
        centeredSlides={true}
        autoplay={{
          delay: 5500,
          disableOnInteraction: false,
        }}
        pagination={{
          clickable: true,
        }}
        modules={[Autoplay, Pagination]}
        className="mySwiper"
        style={{ height: '100vh', color: 'white' }}
      >
      [image1, image2, image1].map((image) => {
      <SwiperSlide>
          <Box sx={{ height: '50vh' }}>
            <Img alt="image" src={image} />
          </Box>
          <Box
            sx={{
              maxWidth: '680px',
              maxHeight: '171px',
              marginLeft: '100px',
              marginTop: '80px',
            }}
          >
            <Typography variant="h4">Welcome!</Typography>
            <Typography>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt ut labore et dolore magna aliqua.
            </Typography>
          </Box>
        </SwiperSlide>
      })
      </Swiper>
```

File: client/src/onePage/AboutUs/AboutUsTexts/AboutUsTexts.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        marginLeft: '80px',
+        color: 'white',
+      }}
+    >
+      <div className="text position-left">
+        <p>Surprising</p>
+        <p>and playful ways</p>
+        <p>to tell a story!</p>
+      </div>
+    </div>
```
Comment (adam-kostuch) [inline client/src/onePage/AboutUs/AboutUsTexts/AboutUsTexts.tsx:21]: Maybe MUI components will be better to use, to be more consistent,

File: client/src/RegisterPage/CustomSlider.tsx (hunk 4 above)
Comment (kinga-kus) [reply client/src/RegisterPage/CustomSlider.tsx]: I forgot to delete this file bc we won't be using a slider. My bad!

File: client/src/onePage/AboutUs/AboutUsTexts/AboutUsTexts.tsx (hunk 5 above)
Comment (kinga-kus) [reply client/src/onePage/AboutUs/AboutUsTexts/AboutUsTexts.tsx:21]: Yeah, I have some idea with this by using a GSAP. More about this later!

**Pull request merged**